string-manipulation-1/src/is-vowel.js

```diff
@@ -0,0 +1,35 @@
+/* exported isVowel */
+function isVowel(char) {
+  var upperCaseVowels = ['A', 'E', 'I', 'O', 'U'];
+  var lowerCaseVowels = ['a', 'e', 'i', 'o', 'u'];
```

Comment:
If you just convert the argument character to either uppercase or lowercase, you can make it so you only need to check one or the other, not both.